packages/steps/src/BuildConfig.ts

```diff
@@ -390,15 +391,15 @@ export function mergeConfigWithImportedFunctions(
 }
 
 export function isBuildStepCommandRun(step: BuildStepConfig): step is BuildStepCommandRun {
-  return typeof step === 'object' && typeof step.run === 'object';
+  return step !== null && typeof step === 'object' && typeof step.run === 'object';
```

Comment:
Haven't checked — can `step` be undefined? Should we loosen the comparison to `!=` or `Boolean(step)`?